deployment/terraform/create.go

```diff
@@ -561,6 +561,10 @@ func (t *Terraform) updateAppConfig(siteURL string, sshc *ssh.Client, jobServerE
 	// Setting to * is more of a quick fix. A proper fix would be to get the DNS name of the first
 	// node or the proxy and set that.
 	cfg.ServiceSettings.AllowCorsFrom = model.NewString("*")
+	cfg.ServiceSettings.EnableOpenTracing = model.NewBool(false)    // Large overhead, better to disable
```

Comment:
This is also set to false by default.

Reply:
This was more a safety measure, in case it is enabled by default at some point, but we can remove it if you have a strong opinion.